src/app/my-messages/community-deleted-message-card/community-deleted-message-card.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DeletableNotificationComponent } from '../deletable-notification/deletable-notification.component';
+import { MessagesService } from '../messages.service';
+import { MatDialog } from '@angular/material';
+import { CommunityDeletedNotification } from './community-deleted-notification';
+
+@Component({
+  selector: 'app-community-deleted-message-card',
+  templateUrl: './community-deleted-message-card.component.html',
+  styleUrls: ['./community-deleted-message-card.component.scss', '../my-messages.component.scss']
```

Comment:
We could use `@import` directive to import `my-messages.component.scss` straight into `community-deleted-message-card.component.scss`.